addons/repos/core.yaml

```diff
@@ -16,6 +16,6 @@ packages:
   - name: pinniped
     versions:
       - 0.12.1
-  - name: local-path-storage
```

Comment:
_note: now that kind deploys this by default and minikube has its own storage provider, no need to include this in our core package repo for unmanaged-clusters_